.changeset/four-kids-wave.md

```diff
@@ -0,0 +1,6 @@
+---
+"@commercetools-docs/gatsby-theme-api-docs": patch
+"@commercetools-website/api-docs-smoke-test": patch
```

Comment:
Don't we just want to publish changes from @commercetools-docs ?

Reply:
yes, thank you.  We have an issue open to configure changeset better